frontend/src/views/Sponsor.vue

```diff
@@ -46,17 +48,11 @@
       <div style="clear:both;"></div>
     </v-container>
     <SponsorModal v-model="dialog" v-bind:title="currentSponsor.name" v-bind:body="currentSponsor.detail"> </SponsorModal>
-
-    <h1 class="text-center text-h1 font-weight-bold">Become a <br> Sponsor</h1>
-    <v-card flat tile style="margin-left:15%">
-      <MailingForm type="sponsorship"></MailingForm>
-    </v-card>
   </div>
 </template>
 
 <script type="text/javascript">
 import SponsorModal from '@/components/SponsorModal';
-import MailingForm from '@/components/MailingForm';
```

Comment:
I see you are removing the MailingForm component. Did you want the API keys + account access to mailjet or are we removing this altogether?

Reply:
We are just temporarily getting rid of it, but since it is part of the requirements for the finshed product we would most likely be just adding it back, but at a later point in time.

Reply:
Sounds good, I guess there should be a general transfer of external accounts and API keys for your teams use. If there's no rush, I'll get that sorted for you guys on Saturday.